test/test_filesystem.cpp

```diff
@@ -18,60 +18,79 @@
 #include "c_utilities/filesystem.h"
 
 static char cwd[1024];
+#ifdef WIN32
+const char delimiter = '\\';
+#else
+const char delimiter = '/';
+#endif
 
 TEST(test_filesystem, utilities_exists) {
   utilities_get_cwd(cwd, 1024);
```

Comment:
Good point, added in https://github.com/ros2/c_utilities/pull/2/commits/87873ccd7555e93e397b04ce2522b529c63c1d57